admission/Makefile

```diff
@@ -19,7 +19,7 @@ crds:
 	curl -fsL -o config/crd/bases/calico.yaml https://docs.projectcalico.org/v$(CALICO_VERSION)/manifests/crds.yaml
 	curl -fsL -o config/crd/bases/contour.yaml https://raw.githubusercontent.com/projectcontour/contour/v$(CONTOUR_VERSION)/examples/contour/01-crds.yaml
 	curl -fsL -o config/crd/bases/application.yaml https://raw.githubusercontent.com/argoproj/argo-cd/v$(ARGOCD_VERSION)/manifests/crds/application-crd.yaml
-	curl -fsL -o config/crd/bases/grafanadashboard.yaml https://github.com/grafana-operator/grafana-operator/raw/v$(GRAFANA_OPERATOR_VERSION)/deploy/crds/GrafanaDashboard.yaml
+	curl -fsL -o config/crd/bases/grafanadashboard.yaml https://github.com/grafana-operator/grafana-operator/raw/v$(GRAFANA_OPERATOR_VERSION)/deploy/manifests/latest/crds.yaml
```

Comment:
The location of CRDs in grafana-operator repo has been changed a few times. 
It looks OK to install all the CRDs to the test environment, however the local filename should be renamed to one other than `grafanadashboard.yaml`.

Reply:
fixed! 68827f2